Dockerfile

```diff
@@ -1,7 +1,7 @@
 #
 # Build
 #
-FROM node:18-slim AS builder
+FROM node:18.19-slim AS builder
```

Comment:
I think we should do this across all our repos, but first we need to configure dependabot to also update `docker` ecosystem packages too and see how it behaves.

Reply:
I've added `docker` to dependabot https://github.com/mozilla/addons-frontend/commit/87a6e1d950240b2b263c0757027f562e5e7570b0